site/app/models/gradeable/AutoGradedVersion.php

```diff
@@ -183,6 +185,51 @@ private function loadSubmissionFiles() {
             }
         }
     }
+    public function getDockerErrorFileExist(): bool {
```

Comment:
Let's clean up the grammar here...

```suggestion
    public function dockerErrorFileExists(): bool {
```